geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl

```diff
@@ -6,7 +6,7 @@ Read the `Documentation <http://docs.camptocamp.net/c2cgeoportal/>`_
 Checkout
 --------
 
-.. code::
+.. script::
```

Comment:
Why this change?

Reply:
You asked me to change it in the demo, no?

Reply:
To remove the prompt, but it's already done :-)

Reply:
OK, reverted